src/subscription_manager/managercli.py

```diff
@@ -53,6 +55,38 @@
 
 log = logging.getLogger(__name__)
 
+conf = config.Config(rhsm.config.initConfig())
+
+SM = "subscription-manager"
+
+
+class SubscribeCommand(AttachCommand):
+    def __init__(self):
+        super(SubscribeCommand, self).__init__()
+
+    def _short_description(self):
+        return _("Deprecated, see attach")
+
+    def _command_name(self):
+        return "subscribe"
+
+    def _primary(self):
+        return False
+
+
+class UnSubscribeCommand(RemoveCommand):
```

Comment:
Same comment for this alias?

Reply:
Same for Subscribe/UnSubscribe - wasn't sure so thought I would leave it and let you tell me to remove it.  Removing.